dashboard.csss-ci.com/server/api/models/Bankrolls_model.php

```diff
@@ -7,18 +7,25 @@ class Bankrolls_model extends CI_Model {
 
     protected $table = "bankrolls";
 
-    // Inserts the new model into the bankrolls table.
-    // 
-    // @data - new bankroll model
-    // @return bool - True if inserted successfully; False, otherwise
+    /*
```

Comment:
Could you also add an extra * to the beginning of the comment? (Replace /* with /**)